docs/guide/api-environment.md

```diff
@@ -68,7 +68,7 @@ export default {
 }
 ```
 
-When not explicitly documented, environment inherits the configured top-level config options (for example, the new `server` and `edge` environments will inherit the `build.sourcemap: false` option). A small number of top-level options, like `optimizeDeps`, only apply to the `client` environment, as they don't work well when applied as a default to server environments. The `client` environment can also be configured explicitly through `environments.client`, but we recommend to do it with the top-level options so the client config remains unchanged when adding new environments.
+When not explicitly documented, environment inherits the configured top-level config options (for example, the new `ssr` and `edge` environments will inherit the `build.sourcemap: false` option). A small number of top-level options, like `optimizeDeps`, only apply to the `client` environment, as they don't work well when applied as a default to server environments. The `client` environment can also be configured explicitly through `environments.client`, but we recommend to do it with the top-level options so the client config remains unchanged when adding new environments.
```

Comment:
`ssr` environment exists implicitly without declaring (explained as "The `client` and a server environment named `ssr` are always present during dev." below). So I think "the new `ssr`" is a bit wrong here.
Probably we can remove the word "new".

Reply:
The word "new" is a little misleading, but `ssr` doesn't build unless at least an empty object is defined. Got it, I'll see what to do, maybe even revert this to `server` + `edge`?